infrastructure/docker/docker-compose.yml

```diff
@@ -1,7 +1,9 @@
 version: '3'
 
 volumes:
-    mysql-data: {}
+    postgres-data: {}
+    # Uncomment to use MySQL
+    # mysql-data: {}
```

Comment:
As a template I don't like it to be polluted by things I don't use.

Reply:
OK, I updated the PR, with a diff and some doc to switch back